ios/NeuroidReactnativeSdk.swift

```diff
@@ -76,6 +76,12 @@ class NeuroidReactnativeSdk: NSObject {
         resolve(true)
     }
 
+    @objc(setRegisteredUserID:withResolver:withRejecter:)
+    func setRegisteredUserID(userID: String, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
+        try? NeuroID.setRegisteredUserID(userID)
```

Comment:
I think we want to return the value from this function don't we? Because they need to know if it failed or not and we already return a true/false it isn't a try/catch anymore